front/app/containers/Navbar/components/NotificationMenu/components/Notification/index.tsx

```diff
@@ -288,6 +290,12 @@ export default class Notification extends PureComponent<Props> {
             }
           />
         );
+      case 'inappropriate_content_flagged':
+        return (
+          <NlpFlaggedPostNotification
+            notification={notification as INlpFlaggedPostNotificationData}
```

Comment:
I suggest you open the branch locally and delete it to see why :-) Not that I don't want to explain, but I think it's more efficient that way.

I do think though, that it's not ideal to pass notification as a prop in NotificationMenu. It would be better to just pass the notification id and make the request in the notification itself. Then we wouldn't need these. I'm not sure if it would impact performance (i.e. needs to make more requests), would have to check that :)